DESCRIPTION

```diff
@@ -23,7 +23,9 @@ Depends:
     R (>= 3.6.0)
 Imports:
     methods,
-    S4Vectors
+    S4Vectors,
+    stats,
+    MASS
```

Comment:
I know `MASS` is a recommended package but would you mind to put it under `Suggests:` instead? (with `requireNamespace` in `robustSummary`)

Reply:
Do we need `requireNamespace` at all, given that it is always available and I call the function with the `MASS::` prefix?